added triviallyCopableJLBH test

j4sm1ne96 opened the pull request:
into develop rather than ea

j4sm1ne96 committed:
added triviallyCopableJLBH test

rogersimmons commented:
Looks good. Pls hold merging for now. Have a couple of small edits pending

JerryShea commented on src/test/java/net/openhft/chronicle/wire/TriviallyCopyableJLBH.java:
```diff
@@ -10,7 +10,6 @@
 import static net.openhft.chronicle.core.pool.ClassAliasPool.CLASS_ALIASES;
-import static net.openhft.chronicle.wire.Marshallable.fromString;
 
 public class TriviallyCopyableJLBH implements JLBHTask {
```
Please add this JLBH to the `run-benchmarks` profile and add 
```
    @Override
    public void complete() {
        TeamCityHelper.teamCityStatsLastRun(this.getClass().getSimpleName(), jlbh, config.iterations, System.out);
    }
```
or similar

Commit:
Add -Dio.type and -Dcpu options
Pin cpu
Use native memory (vs heap)
Reduce throughput (more typical), raise iterations (more percentiles)

rogersimmons commented:
Edits mentioned above now committed

peter-lawrey committed:
Merge branch 'develop' into triviallyCopableJLBH
